<?xml version="1.0" encoding="UTF-8"?><testrun name="TestOneDimTableNatural" project="BT-Library-Java" tests="9" started="9" failures="0" errors="5" ignored="0">
  <testsuite name="Bt.Tests.Core.Collections.TestOneDimTableNatural" time="0.0">
    <testcase name="testCTOR_and_Equal" classname="Bt.Tests.Core.Collections.TestOneDimTableNatural" time="0.0"/>
    <testcase name="test_Insert_ToBigTable_Internal" classname="Bt.Tests.Core.Collections.TestOneDimTableNatural" time="0.0">
      <error>java.lang.ArrayIndexOutOfBoundsException: 5&#13;
	at Bt.Core.ABC.OneDimTable.insert(OneDimTable.java:74)&#13;
	at Bt.Tests.Core.Collections.TestOneDimTableNatural.test_Insert_ToBigTable_Internal(TestOneDimTableNatural.java:51)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at junit.framework.TestCase.runTest(TestCase.java:176)&#13;
	at junit.framework.TestCase.runBare(TestCase.java:141)&#13;
	at junit.framework.TestResult$1.protect(TestResult.java:122)&#13;
	at junit.framework.TestResult.runProtected(TestResult.java:142)&#13;
	at junit.framework.TestResult.run(TestResult.java:125)&#13;
	at junit.framework.TestCase.run(TestCase.java:129)&#13;
	at junit.framework.TestSuite.runTest(TestSuite.java:252)&#13;
	at junit.framework.TestSuite.run(TestSuite.java:247)&#13;
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:86)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</error>
    </testcase>
    <testcase name="test_Insert_ToBigTable_LeftLimit" classname="Bt.Tests.Core.Collections.TestOneDimTableNatural" time="0.0">
      <error>java.lang.ArrayIndexOutOfBoundsException: 5&#13;
	at Bt.Core.ABC.OneDimTable.insert(OneDimTable.java:74)&#13;
	at Bt.Tests.Core.Collections.TestOneDimTableNatural.test_Insert_ToBigTable_LeftLimit(TestOneDimTableNatural.java:77)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at junit.framework.TestCase.runTest(TestCase.java:176)&#13;
	at junit.framework.TestCase.runBare(TestCase.java:141)&#13;
	at junit.framework.TestResult$1.protect(TestResult.java:122)&#13;
	at junit.framework.TestResult.runProtected(TestResult.java:142)&#13;
	at junit.framework.TestResult.run(TestResult.java:125)&#13;
	at junit.framework.TestCase.run(TestCase.java:129)&#13;
	at junit.framework.TestSuite.runTest(TestSuite.java:252)&#13;
	at junit.framework.TestSuite.run(TestSuite.java:247)&#13;
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:86)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</error>
    </testcase>
    <testcase name="test_Insert_ToBigTable_PreviousOfLast" classname="Bt.Tests.Core.Collections.TestOneDimTableNatural" time="0.0">
      <error>java.lang.ArrayIndexOutOfBoundsException: 5&#13;
	at Bt.Core.ABC.OneDimTable.insert(OneDimTable.java:74)&#13;
	at Bt.Tests.Core.Collections.TestOneDimTableNatural.test_Insert_ToBigTable_PreviousOfLast(TestOneDimTableNatural.java:129)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at junit.framework.TestCase.runTest(TestCase.java:176)&#13;
	at junit.framework.TestCase.runBare(TestCase.java:141)&#13;
	at junit.framework.TestResult$1.protect(TestResult.java:122)&#13;
	at junit.framework.TestResult.runProtected(TestResult.java:142)&#13;
	at junit.framework.TestResult.run(TestResult.java:125)&#13;
	at junit.framework.TestCase.run(TestCase.java:129)&#13;
	at junit.framework.TestSuite.runTest(TestSuite.java:252)&#13;
	at junit.framework.TestSuite.run(TestSuite.java:247)&#13;
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:86)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</error>
    </testcase>
    <testcase name="test_Insert_ToBigTable_RightLimit" classname="Bt.Tests.Core.Collections.TestOneDimTableNatural" time="0.0">
      <error>java.lang.ArrayIndexOutOfBoundsException: 5&#13;
	at Bt.Core.ABC.OneDimTable.insert(OneDimTable.java:74)&#13;
	at Bt.Tests.Core.Collections.TestOneDimTableNatural.test_Insert_ToBigTable_RightLimit(TestOneDimTableNatural.java:155)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at junit.framework.TestCase.runTest(TestCase.java:176)&#13;
	at junit.framework.TestCase.runBare(TestCase.java:141)&#13;
	at junit.framework.TestResult$1.protect(TestResult.java:122)&#13;
	at junit.framework.TestResult.runProtected(TestResult.java:142)&#13;
	at junit.framework.TestResult.run(TestResult.java:125)&#13;
	at junit.framework.TestCase.run(TestCase.java:129)&#13;
	at junit.framework.TestSuite.runTest(TestSuite.java:252)&#13;
	at junit.framework.TestSuite.run(TestSuite.java:247)&#13;
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:86)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</error>
    </testcase>
    <testcase name="test_Insert_ToBigTable_Second" classname="Bt.Tests.Core.Collections.TestOneDimTableNatural" time="0.0">
      <error>java.lang.ArrayIndexOutOfBoundsException: 5&#13;
	at Bt.Core.ABC.OneDimTable.insert(OneDimTable.java:74)&#13;
	at Bt.Tests.Core.Collections.TestOneDimTableNatural.test_Insert_ToBigTable_Second(TestOneDimTableNatural.java:103)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at junit.framework.TestCase.runTest(TestCase.java:176)&#13;
	at junit.framework.TestCase.runBare(TestCase.java:141)&#13;
	at junit.framework.TestResult$1.protect(TestResult.java:122)&#13;
	at junit.framework.TestResult.runProtected(TestResult.java:142)&#13;
	at junit.framework.TestResult.run(TestResult.java:125)&#13;
	at junit.framework.TestCase.run(TestCase.java:129)&#13;
	at junit.framework.TestSuite.runTest(TestSuite.java:252)&#13;
	at junit.framework.TestSuite.run(TestSuite.java:247)&#13;
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:86)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</error>
    </testcase>
    <testcase name="test_Insert_ToEmpty" classname="Bt.Tests.Core.Collections.TestOneDimTableNatural" time="0.0"/>
    <testcase name="test_Insert_ToSingleEl_ElHigher" classname="Bt.Tests.Core.Collections.TestOneDimTableNatural" time="0.0"/>
    <testcase name="test_Insert_ToSingleEl_ElLower" classname="Bt.Tests.Core.Collections.TestOneDimTableNatural" time="0.0"/>
  </testsuite>
</testrun>
